db/db.changelog-master.xml

```diff
@@ -23,5 +23,6 @@
     <include file="v-2/tables/db.changelog-tables.xml" relativeToChangelogFile="true"/>
     <include file="test-data/db.changelog-insert.xml" relativeToChangelogFile="true" context="dev"/>
     <include file="after-all-migrations/db.changelog-after-all.xml" relativeToChangelogFile="true" context="dev or prod or test"/>
+    <include file="sandbox/create-sandbox-schema.xml" relativeToChangelogFile="true" />
```

Comment:
it should be separated from main changelog

Reply:
I added it for testing. But I think that the scheme must be created by ourselves in the database. Not through Liquidbase.

Reply:
actually, we were discussing it with @akuleshov7. 
as I remember, it was agreed that having such database change as liquibase script it good. (@akuleshov7 correct me, if I mistake)
so I think it's okay to keep this script

Reply:
I think that it is not correct that a new schema will be created with each test and rollover of the base. It seems to me that the schema, as part of the database, must be created manually.